pkg/models/live.go

```diff
@@ -0,0 +1,30 @@
+package models
+
+import "github.com/centrifugal/centrifuge"
+
+// ChannelPublisher writes data into a channel
+type ChannelPublisher func(channel string, data []byte) error
```

Comment:
@kaydelaney -- this was my alternative to what you did with an async thread on HTTPServer

I don't think we should expose this on HTTPServer, but we should consider exposing this as as a go chan